elfi/visualization/visualization.py

```diff
@@ -359,3 +359,42 @@ def plot_params_vs_node(node, n_samples=100, func=None, seed=None, axes=None, **
             axes[idx].set_axis_off()
 
     return axes
+
+
+def plot_discrepancy(gp, parameter_names, axes=None, **kwargs):
+    """Plot acquired parameters vs. resulting discrepancy.
+
+    Parameters
+    ----------
+    axes : matplotlib.axes.Axes, optional
+    gp : GPyRegression target model, required
+    parameter_names : dict, required
+        Parameter names from model.parameters dict('parameter_name':(lower, upper), ... )`
+    axes : matplotlib.axes.Axes, optional
```

Comment:
axes : matplotlib.axes.Axes, optional 
appears twice
also:
axes : matplotlib.axes.Axes, optional
or
axes : np.array of plt.Axes ?